src/Leaves/Controls/Control.php

```diff
@@ -44,6 +44,11 @@ public function getValue()
         return $this->model->value;
     }
 
+    public function getHtmlId()
```

Comment:
I think this should maybe move up to the Leaf class as it's generally useful. getPath() maybe as a counterpart to getName() which already exists? Out of interest why did the parent class need this? Just wondering if there's a scope issue?